tasktree/core/__init__.py

```diff
@@ -0,0 +1,49 @@
+"""Core module of task_tree project."""
+
+from random import choice
+from anytree import NodeMixin
```

Comment:
Specify dependencies in requirements.txt

Reply:
In which subdirectory to store the requirements.txt?